mbed_flasher/flashers/__init__.py

```diff
@@ -15,13 +15,17 @@
 limitations under the License.
 """
 
-from mbed_flasher.flashers.FlasherMbed import FlasherMbed as mbed_flasher
+from mbed_flasher.flashers.FlasherMbed import FlasherMbed
+from mbed_flasher.flashers.FlasherJLink import FlasherJLink
 
 # disable Invalid constant name warning, not a const
 # pylint: disable=C0103
-AvailableFlashers = [
```

Comment:
I remember this namespace changes seems were for some tests,  maybe here will have some errors.